src/main/java/team/gdsc/code2cv/core/jwt/JwtProvider.java

```diff
@@ -1,125 +1,125 @@
-                .accessToken(accessToken)
-                .refreshToken(refreshToken)
-                .build();
-    }
-
-    /**
-     * Jwt가 유효한지 검사하는 메서드.
-     * 만료시간, 토큰의 유효성을 검사한다.
-     */
-    public boolean validateToken(String rawToken) {
-        try {
-            Claims claims = extractClaims(rawToken);
-            return !claims.getExpiration().before(new Date());
-        } catch (Exception e) {//JwtException, ExpiredJwtException, NullPointerException
-            return false;
-        }
-    }
-
-    /**
-     * [validateToken] 이후 호출하는 메서드.
-     * refreshToken을 통해, accessToken을 재발급하는 메서드.
-     * refreshToken의 유효성을 검사하고, isAccessToken이 true일때만 accessToken을 재발급한다.
-     */
-    public String reissueAccessToken(String refreshToken) {
-        Claims claims = extractClaims(refreshToken);
-        if (claims.get(IS_ACCESS_TOKEN, Boolean.class)) {
-            throw new JwtException("RefreshToken이 유효하지 않습니다.");
-        }
-        JwtUser jwtUser = claimsToJwtUser(claims);
-        return generateToken(jwtUser, true);
-
-    }
-
-    /**
-     * [validateToken] 이후 호출하는 메서드.
-     * rawToken을 통해 JwtUser를 추출한다.
-     * [jwtUser]는 userId와 role을 가지고 있다. 즉 JWT에 저장된 정보를 추출한다.
-     */
-    public JwtUser getJwtUser(String rawToken) {
-        Claims claims = extractClaims(rawToken);
-        return claimsToJwtUser(claims);
-    }
-
-    private JwtUser claimsToJwtUser(Claims claims) {
-        Role userRole = Role.valueOf(claims.get(ROLE, String.class));
-        String userId = claims.getSubject();
-        return JwtUser.of(Long.parseLong(userId), userRole);
-    }
-
-
-    /**
-     * Jwt 토큰생성
-     * accessToken과 refreshToken의 다른점은 만료시간과, isAccessToken이다.
-     */
-    private String generateToken(JwtUser jwtUser, boolean isAccessToken) {
-        Key secretKey = generateKey();
-        long expireTime = isAccessToken ? ACCESS_TOKEN_EXPIRE_TIME : REFRESH_TOKEN_EXPIRE_TIME;
-        Date expireDate = new Date(System.currentTimeMillis() + expireTime);
-        return Jwts.builder()
-                .signWith(secretKey)
-                .claim(ROLE, jwtUser.getRole().getValue())
-                .claim(IS_ACCESS_TOKEN, isAccessToken)
-                .setSubject(jwtUser.getId().toString())
-                .setExpiration(expireDate)
-                .compact();
-    }
-
-
-    /**
-     * HS256방식의 키를 생성한다.
-     */
-    private Key generateKey() {
-        return new SecretKeySpec(secret.getBytes(), SignatureAlgorithm.HS256.getJcaName());
-    }
-
-    private Claims extractClaims(String rawToken) {
-        return Jwts.parserBuilder()
-                .setSigningKey(generateKey())
-                .build()
-                .parseClaimsJws(rawToken)
-                .getBody();
-    }
+	@Value("${jwt.secret}")
+	private String secret;
+
+	@Value("${jwt.access-token-expire-time}")
+	private long ACCESS_TOKEN_EXPIRE_TIME;
+	@Value("${jwt.refresh-token-expire-time}")
+	private long REFRESH_TOKEN_EXPIRE_TIME;
```

Comment:
```suggestion
	@Value("${jwt.access-token-expire-time}")
	private long accessTokenExpireTime;
	@Value("${jwt.refresh-token-expire-time}")
	private long refreshTokenExpireTime;
```

상수 느낌으로 사용하기때문에 대문자로 작성한 점을 알고있지만, 이렇게 사용하는 코드도 많이 봤고 크게 문제없을 것 같아요.